tests/conftest.py

```diff
@@ -235,20 +235,13 @@ def __init__(self, name=None, imgname=None, keeptb=False, fakeplatform=None):
         self.redis_sock = self.mount + '/' + "redis.sock"
         self.check_ctn_status_and_db_connect()
 
-        # Initialize DB Wrappers
-        self.app_db = dvs_db.DVSDatabase(self.APP_DB_ID, self.redis_sock)
-        self.asic_db = dvs_db.DVSDatabase(self.ASIC_DB_ID, self.redis_sock)
-        self.counters_db = dvs_db.DVSDatabase(self.COUNTERS_DB_ID, self.redis_sock)
-        self.config_db = dvs_db.DVSDatabase(self.CONFIG_DB_ID, self.redis_sock)
-        self.flex_db = dvs_db.DVSDatabase(self.FLEX_COUNTER_DB_ID, self.redis_sock)
-        self.state_db = dvs_db.DVSDatabase(self.STATE_DB_ID, self.redis_sock)
-
-        # NOTE: In order to limit the scope of this PR only the newly ported
-        # tests are using these variables. I will be refactoring the tests to
-        # all use these variables in a follow-up PR.
-        self.default_acl_tables = self.asicdb.default_acl_tables
-        self.default_acl_entries = self.asicdb.default_acl_entries
-        self.port_name_map = self.asicdb.portnamemap
+        # DB wrappers are declared here, lazy-loaded in the tests
```

Comment:
I am not sure if there would be some runtime hit due to the lazy loading. if yes, you could still initialize all of them here and just return the handles in the get_*_db wrapper functions

Reply:
I don't think there's really a big difference either way.

The main reason I opted to lazy-load the DBs is that a lot of the tests are only using a couple of them anyways, so there's no need to create all 6 every time. I think ideally we'll move towards setting up the needed dbs in some sort of module-level fixture or something so that the author doesn't have to remember to include `setup_db` for each test case.